content/languages/python/codewars-test/index.md

```diff
@@ -223,6 +216,19 @@ test.expect_no_error(msg, f, OSError)          # Pass
 
 ## Timeout Utility
 
+Runs the decorated function within the time limit.  
+
+`sec` is the amount of time allowed. It is expressed in seconds and can be given as an integer or float.  
+Generates a failed assertion when the function fails to complete in time, and its execution is terminated immediately.
+
+Decorated function is required to not throw any exception. If an error is raised during its execution, the test is considered failed and the error message becomes _"Should not throw any exceptions inside timeout: \<Exception()\>"_. This requirement is enforced by wrapping the inner function with `expect_no_error`, and as a side effect, one extra "test passed" entry is emitted for a collection of tests run inside a timeout wrapper.
+
+:::warning
+Timed test should contain at least one assertion which verifies the result returned by the user solution. Otherwise, the test will be considered passed just if it happens to finish in time below the requested time limit, even if it would return incorrect answer.
```

Comment:
```suggestion
Timed tests should contain at least one assertion which verifies the result returned by the user solution. Otherwise, the test will be considered passed just if it happens to finish in time below the requested time limit, even if it would return an incorrect answer.
```